src/components/dashboard-settings/sync-apps/sync-apps.directive.js.coffee

```diff
@@ -66,6 +67,8 @@ module.directive('dashboardSettingSyncApps', ($templateCache, $log, $http, $filt
         )
         scope.hasError = scope.connectors && scope.connectors.length > 0 && _.any(scope.connectors, (c) -> isError(c))
 
+        scope.has_running_apps = scope.connectors.length > 0 && scope.has_running_cube
```

Comment:
shouldn't it be `|| scope.has_running_cube` ? (= has connectors OR has cubes)

Reply:
That should totally be an OR